src/player.cpp

```diff
@@ -219,6 +221,12 @@ void Player::Run() {
 	// libretro invokes the MainLoop through a retro_run-callback
 #else
 	while (Transition::instance().IsActive() || (Scene::instance && Scene::instance->type != Scene::Null)) {
+#if defined(__WIIU__)
```

Comment:
I had these applet lifetime checks moved from here for the 3DS and switch ports, no real need to add a platform dependency here.
See also https://github.com/EasyRPG/Player/commit/c9b1809898bfb687cd2e775710187887ec146729#diff-10c39dd978ada90e0d090a045c4ae5f51dad010f3b754679e15894c9b93d254b